### PR TITLE
Hott-203 Adds a CDN to the API service

### PR DESCRIPTION
**Pull request by mafonso**
What:
Creates a CDN for the API service, with configuration per environment
Creates and validates the respective ACM certificate.

Why:
We need to take over this CDN into our account.
We want this to be persisted in code

### CODE REVIEW

**Review by willfish — APPROVED** [review]

File: terraform/api-docs/vars/production.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+environment_name="production"
+environment_key="production"
+fqdn="api.trade-tariff.service.gov.uk"
```
Comment (willfish) [inline terraform/api-docs/vars/production.tf:3]:
```suggestion
fqdn="api2.trade-tariff.service.gov.uk"
```

File: terraform/api-docs/vars/production.tf [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+aliases = [ "api.trade-tariff.service.gov.uk"]
```
Comment (tavy88) [inline terraform/api-docs/vars/production.tf:4]: Extra space here.

Comment (willfish) [inline terraform/api-docs/vars/production.tf:4]:
```suggestion
aliases = ["api.trade-tariff.service.gov.uk"]
```

File: terraform/api-docs/vars/development.tf [hunk 3]
```diff
@@ -0,0 +1,5 @@
+environment_name="development"
+environment_key="dev"
+fqdn="api-dev.trade-tariff.service.gov.uk"
+aliases = [ ]
```
Comment (tavy88) [inline terraform/api-docs/vars/development.tf:4]: Is this supposed to be empty?

File: terraform/api-docs/vars/production.tf (hunk 2 above)
Comment (mafonso) [reply terraform/api-docs/vars/production.tf]: Actually there is another issue.  That alias matches the fqdn and need to be removed (it's was on dev).
While not a problem and totally valid it causes the certificate to always be recreated every time.

File: terraform/api-docs/vars/development.tf (hunk 3 above)
Comment (mafonso) [reply terraform/api-docs/vars/development.tf:4]: Yes, we don't need aliases here, but need the option.

**Pull request merged**